docs/pages/getting-started/3-configure-your-node.rst

```diff
@@ -11,65 +11,66 @@ Prerequisites
 
-Consult the community on `Slack <https://nuts-foundation.slack.com/>`_ to find out about how to connect to either network.
+There are 3 networks:
 
-Configuring
-***********
+- `development` where new features are tested. Nodes will generally run the newest (not yet released) version of the Nuts node.
+- `stable` for integrating your software with Nuts and testing with other vendors. Nodes will generally run the latest released version (or at least a recent one).
+- `production` for production uses. Connecting to this network involves PKIoverheid certificates and outside the cope of this tutorial.
 
-1. Configure the bootstrap nodes using ``network.bootstrapnodes``.
-2. Configure TLS using ``network.certfile``, ``network.certkeyfile`` and ``network.truststorefile``.
+Node TLS Certificate
+====================
 
-See :ref:`configuration reference <nuts-node-config>` for a detailed explanation on how to exactly configure the Nuts node.
+Before you can join a network, your node needs a certificate from the correct Certificate Authority (CA). The two `development` and`stable` networks are open for everyone to join. Contrary to the `production` network (where we will be using a real Certificate Authority like PKIoverheid) the CA certificate and private key for these networks are available on github. This way you can generate your own certificate.
 
+To generate the certificate for your own node you need the ``https://github.com/nuts-foundation/nuts-development-network-ca`` repository. It contains handy scripts and the needed key material.
 
-.. note::
+Your node only accepts requests from other nodes which use a certificate issued by one of the trusted CAs. Which CAs to trust is configured by providing a truststore file. The truststore is a PEM file which contains a set of one or more certificates from CAs which the network participants all decided on to trust.
+To learn more about how a Nuts network uses certificates, see the specification `RFC008 <https://nuts-foundation.gitbook.io/drafts/rfc/rfc008-certificate-structure>`_.
 
-    You can start the node without configuring the network, but it won't connect and thus exchange data with other nodes.
 
-YAML Configuration File
-=======================
+To generate certificates for the `development` network perform the following steps:
 
-If you're using a YAML file to configure your node, the following snippet shows an example for the network related configuration:
+.. code-block:: shell
 
-.. code-block:: yaml
+  git clone https://github.com/nuts-foundation/nuts-development-network-ca
+  cd nuts-development-network-ca
+  ./issue-cert.sh development nuts.yourdomain.example
 
-  network:
-    truststorefile: /path/to/truststore.pem
-    certfile: /path/to/certificate-and-key.pem
-    certkeyfile: /path/to/certificate-and-key.pem
-    bootstrapnodes:
-      - example.com:5555
+This results in 3 files:
```

Comment:
has the script been changed? The CA had to be copied from the ca directory manually.

Reply:
what about pointing to the README of that repo? Now both have to be kept in sync.

Reply:
The script did indeed changed. Now it copies the $NETWORK/ca to ./truststore-$NETWORK.pem. So it is clear what the file contains and for which network.